avro-converter/src/main/java/io/confluent/connect/avro/AvroConverter.java

```diff
@@ -1,18 +1,16 @@
-/**
- * Copyright 2015 Confluent Inc.
+/*
+ * Copyright 2018 Confluent Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Licensed under the Confluent Community License; you may not use this file
```

Comment:
maintain same legal verbiage. Do not remove (the "License")

Reply:
I will let Terry from Legal respond to this and others.

Reply:
Agreed that (the "License") should remain.